django/db/models/sql/query.py

```diff
@@ -1273,6 +1285,41 @@ def _add_q(self, q_object, used_aliases, branch_negated=False,
         needed_inner = joinpromoter.update_join_types(self)
         return target_clause, needed_inner
 
+    def build_filtered_relation_q(
+            self, q_object, force_reuse, branch_negated=False,
+            current_negated=False):
```

Comment:
Unusual style.. should have args starting in the first line and aligned to the `(`.